gbBuild.py

```diff
@@ -1,336 +1,266 @@
+import csv
+import hashlib
+import json
 import os
+import random
+import shutil
 import sys
-import gbHelpers
-import gbDataCheck
+import time
 import zipfile
-import gbMetaCheck
-import csv
-import json
+from datetime import datetime
 from distutils.dir_util import copy_tree
-import time
+from pathlib import Path
+
 import geopandas
-import shutil
-import hashlib
 import matplotlib.pyplot as plt
-from shapely.geometry.polygon import Polygon
-from shapely.geometry.multipolygon import MultiPolygon
-from datetime import datetime
 import requests
+from github import github
+from shapely.geometry.multipolygon import MultiPolygon
+from shapely.geometry.polygon import Polygon
 
-buildType = str(sys.argv[1])
-buildVer = str(sys.argv[2])
-cQuery = str(sys.argv[3])
-typeQuery = str(sys.argv[4])
```

Comment:
are these optional ? or are you running this script like 664 times per buildType? I'm kinda confused because sys.argv raises an exception if these aren't defined but when you are also using os.walk it is always filtering the list of files that it finds. 

With your workflow do you want to change this in any way? for example, maybe it would be more helpful to have typeQuery be an optional argument (eg. `-type ADM1`)? Or we could have both be optional arguments that can include multiple selections via comma-delimiter like you have now only for cQuery ?

If you are always referencing only one source file maybe it makes more sense to combine these arguments into one: `IND_ADM1`

Reply:
we can accommodate any option, but what is the most common use case ? and what is never a desired behavior ?

Reply:
This is because the script predominantly interacts with automated jobs launched from github (see https://github.com/wmgeolab/geoBoundaries/blob/main/.github/workflows/yDISPATCH_gBLayer.yml).  I retained the walk behavior from an older build, which I *think* is enabling multi-layer PRs that get submitted to be validated on the automated PR-response side of things.  But I honestly don't remember if that's still true.